pom.xml

```diff
@@ -209,6 +209,11 @@
             <artifactId>log4j-over-slf4j</artifactId>

```

Comment:
@nguyenq the changes of this file are safe and makes sende for me, though crappy format.